README.md

```diff
@@ -18,12 +18,11 @@ brew install git node llvm@8
 npm i -g yarn
 ```
 
-**Ubuntu 18.04**
+**Debian/Ubuntu**
 
-```
-apt-get update && apt-get install -y git nodejs npm clang libclang-dev llvm
-npm i -g yarn
-```
+1. `apt install clang libclang-dev llvm`
+1. Install Node.js (if it's not already installed): `apt install nodejs npm`
```

Comment:
I'm slightly confused: The PR description says it's not idiomatic to install nodejs and npm via apt, but that's still what we're doing here, right? Did you mean "install yarn via npm" instead?

Separately, a minor nit: I realize markdown processors will render the text with sequential numbers, but part of the point of Markdown is it should be human-readable even in plain text form. I personally dislike the style of using "1." for every bullet, especially when there are only a few. YMMV.

I'd argue the code fence was preferable, because it was copy-pasteable into the terminal. What do you think about something like:

```
apt install clang libclang-dev llvm
if ! which nodejs ; then apt install nodejs npm ; end
``` 

Having a naked pointer to a long page of mixed-system install instructions seems like a regression to me, and if I read it right you may have to click through to at least one additional page (if your Ubuntu is old enough).  Maybe we can just inline the hot path here?

```
curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
sudo apt update && sudo apt install yarn
```